app/components/syntax-highlighted-diff.hbs

```diff
@@ -1,4 +1,12 @@
-<div class="overflow-x-scroll" {{did-update this.handleDidUpdateCode @code}} {{is-dark-mode this.handleIsDarkModeUpdate}} ...attributes>
+<div
+  class="overflow-x-scroll dark:text-gray-400 dark:bg-gray-900"
+  {{did-insert (perform this.highlightCode)}}
+  {{did-update (perform this.highlightCode) this.darkMode.isEnabled}}
```

Comment:
I think that `did-update` takes multiple args, so we could just pass all of them in one go

Reply:
Oh, that's cool if it does now, certainly didn't before, will check!